scripts/backtest_sites.py

```diff
@@ -67,6 +70,10 @@
 # checkpoint on the val set
 model_chckpoint_dir = "PLACEHOLDER"
 
+revision = None
```

Comment:
A small one but maybe these could have `hf_` at the beginning just so it's immediately clear they are hf related variables?

Reply:
yes good idea!